fdbrpc/fdbrpc.h

```diff
@@ -121,6 +121,11 @@ class ReplyPromise final : public ComposedIdentifier<T, 1> {
 	void sendError(const E& exc) const {
 		sav->sendError(exc);
 	}
+	// Call this function when a server will never send a reply to avoid sending
+	// a broken_promise when the ReplyPromise is destroyed
+	void sendNever() const {
```

Comment:
I think if you change this to `void send(Never) const` instead this will sort of "just work" with existing code.